### PR TITLE
data: sabrinalzbh

### PR DESCRIPTION
**Pull request by sabrinalzbh**
~ Socials Update ( provided Github account twice. Added LinkedIn URL).

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6308"></a>

### CODE REVIEW

File: data/sabrinalzbh.json [hunk 1]
```diff
@@ -22,13 +22,12 @@
       "url": "https://github.com/sabrinalzbh/",
       "icon": "FaGithub"
     },
+    {
+      "group": "Socials Primary",
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/sabrina-chacon/",
+      "icon": "FaLinkedIn"
```
Comment (Sindhuinti) [inline data/sabrinalzbh.json:29]: Please change icon name to `FaLinkedin`

**Review by Sindhuinti — CHANGES_REQUESTED** [review]

File: data/sabrinalzbh.json (hunk 1 above)
Comment (sabrinalzbh) [reply data/sabrinalzbh.json:29]: Will do. Thank you. 😌

**Review by Sindhuinti — APPROVED** [review]
Looks good now 🎉

**Comment by sabrinalzbh** [comment]
> Looks good now 🎉 

~ Yaay!! Thank you!

**Comment by SaraJaoude** [comment]
Hi @sabrinalzbh! You have conflicts in your file - this is usually because your fork is not up to date.  

Hopefully the following Docs and video help: https://linkfree.eddiehub.io/docs/how-to-guides/editing

**Comment by sabrinalzbh** [comment]
> Hi @sabrinalzbh! You have conflicts in your file - this is usually because your fork is not up to date.  
> 
> 
> 
> Hopefully the following Docs and video help: https://linkfree.eddiehub.io/docs/how-to-guides/editing

~ Hola @SaraJaoude 😊 I'll give that another shot. Thank you!

**Pull request closed without merging**